<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGenieFrameDebug</class>
 <widget class="QFrame" name="QGenieFrameDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>113</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="pushButton_0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>73</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>10</y>
     <width>73</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>10</y>
     <width>76</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>113</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>10</y>
     <width>73</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>url</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>33</y>
     <width>73</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>messagebox</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>33</y>
     <width>114</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>tray</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>33</y>
     <width>75</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>33</y>
     <width>73</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>33</y>
     <width>73</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>10</y>
     <width>73</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>33</y>
     <width>73</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
